### PR TITLE
Expose the option for specifying operations to exclude

### PR DESCRIPTION
**Pull request by tengqm**
/hold

Wait for #181 to go in first.

### CODE REVIEW

File: gen-apidocs/config/v1_20/config.yaml [hunk 1]
```diff
@@ -271,3 +276,20 @@ operation_categories:
...
       match: read${group}${version}(Namespaced)?${resource}Log
+
+# List of *partial* operation IDs for matching. All matched operations are
+# excluded from the reference doc.
+excluded_operations:
+  - connectCoreV1Patch
+  - createCoreV1NamespacedPodBinding
+  - getCodeVersion
+  - logFileHandler
+  - logFileListHandler
```
Comment (kbhawkey) [inline gen-apidocs/config/v1_20/config.yaml:287]: OK, good. This is removed from the code and is now configurable.

**Comment by kbhawkey** [comment]
I read through the follow-up PRs #182, #183, #184. The configuration updates make sense.

**Comment by tengqm** [comment]
/hold cancel
#181 is in.

**Pull request merged**